### PR TITLE
Feat/create our events section

### PR DESCRIPTION
**Pull request by guilhermefxs**
## What I did

- Create Our Events Section
   - create a react slick carousel to our events, which are events that already happend with cards

- Create a function that gets events and separate between our events and agenda

## How to test

1. Open the terminal and go to the folder of this project

1. In terminal, open 3 tabs:
    1. Tab 1:
       1. `git checkout develop`
       1. `git pull`
       1. `git checkout feat/create-our-events-section
       1. `git pull origin feat/create-our-events-section
    1. Tab 2:
       1. `cd server`
       1. `npm install`
       1. `npm run start`
    1. Tab 3:
       1. `cd client`
       1. `node index.js`

1. In your browser go to http://localhost:3000 and http://localhost:3001/admin

1. Check if the component looks just like the mockup in the erspective widths
    1. 3000px (Larger than usual - Margin has to grow and the content stay in the middle)
    1. 1920px (large web - Has to be axactly equals to me mockup)
    1. 1600px (usual web - the content has to fit the screen without breaking)
    1. 1025px (Smallest web - the content has to fit the screen without breaking)
    1. 1024px (Biggest mobile - the content has to fit the screen without breaking)
    1. 600px (Usual mobile - the content has to fit the screen without breaking)
    1. 320px (Smallest mobile - the content has to fit the screen without breaking)
    
1. In the admin page, try to edit something relative to this issue and check if it updates in the front

1. Read the changed files section on GitHub and check if the code is okay

1. Test the things that you think are worth testing, even the ones that are not in this description

### CODE REVIEW

File: client/src/components/OurEvents/OurEvents.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        centerPadding: "60px",
+        slidesToShow: 1,
+        speed: 500,
+        variableWidth: true
+      };
+
+      
+
+  return (
+    <section className="our-events">
```
Comment (Monterazo) [inline client/src/components/OurEvents/OurEvents.jsx]: Needs to have an id so the navbar can find this section

**Review by Monterazo — CHANGES_REQUESTED** [review]
It works fine, but you could fix the shadows by deleting the .ourEventsCard div margin and by adding 25px to the bottom margin and 12px for left and right margin. Also, the description font-weight should be switched to 401, so the color is the same as in the mockup and the cards get too tall in bigger screens

**Review by Monterazo — APPROVED** [review]
All fixed, working fine!

**Review by raissaheimann — APPROVED** [review]
Everything is perfect after you changed the things Monterazzo asked you for. Congrats :)

**Pull request merged**